tests/integration_tests/performance/test_process_startup_time.py

```diff
@@ -45,3 +46,59 @@ def test_startup_time(test_microvm_with_api):
 
     assert cpu_startup_time_us > 0
     assert cpu_startup_time_us <= MAX_STARTUP_TIME_CPU_US[platform.machine()]
+
+
+def _custom_filter_setup(test_microvm):
+    bpf_path = os.path.join(test_microvm.path, 'bpf.out')
+
+    cargo_target = '{}-unknown-linux-musl'.format(platform.machine())
+    json_path = '../resources/seccomp/{}.json'.format(cargo_target)
+
+    cmd = 'cargo run -p seccomp --target {} -- --input-file {} --target-arch\
+        {} --output-file {}'.format(cargo_target, json_path,
+                                    platform.machine(), bpf_path)
+
+    utils.run_cmd(cmd)
+
+    test_microvm.create_jailed_resource(bpf_path)
+    test_microvm.jailer.extra_args.update({"seccomp-filter": 'bpf.out'})
+
+
+def test_startup_time_custom_seccomp(test_microvm_with_api):
```

Comment:
Do we need a snapshot specific test? Are we sure the codepath is the same for boot/snapshot resume ?

Reply:
We don't need a snapshot specific test. The function that is responsible for opening and deserializing the binary file is called here: https://github.com/firecracker-microvm/firecracker/blob/71be3ded38bf6852c90b8607d4b5276640e6d481/src/firecracker/src/main.rs#L215

This code path is absolutely the same irrespective of what the Firecracker process will do (boot from JSON, boot from API, start from snapshot).